src/main/java/com/revature/ams/util/auth/AuthController.java

```diff
@@ -0,0 +1,36 @@
+package com.revature.ams.util.auth;
+
+import com.revature.ams.Member.Member;
+
+import javax.security.sasl.AuthenticationException;
+import java.util.Scanner;
+
+// TODO: REVIEW ME
+public class AuthController {
+
+    private final Scanner scanner;
+    private final AuthService authService;
+
+    public AuthController(Scanner scanner, AuthService authService) {
+        this.scanner = scanner;
+        this.authService = authService;
+    }
+
+    public Member login(Member memberLoggedIn){
+        try {
+            if(memberLoggedIn != null) throw new RuntimeException("Already logged in");
```

Comment:
would a runtime exception crash the program?

Reply:
No, because instead of ducking the exception we end up handling it directly where it is thrown.